pkg/operator/status/status_controller.go

```diff
@@ -130,6 +131,14 @@ func (c *StatusSyncer) WithVersionRemoval() *StatusSyncer {
 	return &output
 }
 
+// WithEmptyVersionRemoval returns a copy of the StatusSyncer that will
+// remove versions that are an empty string in VersionGetter from the status.
+func (c *StatusSyncer) WithEmptyVersionRemoval() *StatusSyncer {
```

Comment:
could we use `WithVersionRemoval` instead ?

Reply:
mhm, maybe we should allow for version removal then ? 

I think that would involve exposing a new method on https://github.com/openshift/cluster-openshift-apiserver-operator/blob/476eac3b927b2a935f01f8ac4ec74f9a0cf54bc4/vendor/github.com/openshift/library-go/pkg/operator/status/version.go#L33 


If we had such method then we could pair it with WithVersionRemoval, right ?